docs/serving/webhook-customizations.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Webhook Customizations"
+weight: 70
+type: "docs"
+---
+
+## Exclude Namespaces from Webhook
```

Comment:
```suggestion
## Exclude namespaces from the Knative webhook
```

Reply:
Actually I'd rename the doc 
"Webhook Customizations" -> "Exclude namespaces from the Knative webhook"
and then remove this ## heading entirely, since you're not really customizing, just disabling.

Reply:
Good idea. I was trying to set this up as a doc that may include other customizations. Trying to figure out if this is better as a single doc or added to something else. I'll update it like it is always going to be a solo doc and use the title you suggested.

Reply:
Yeah totally know what you mean :slightly_smiling_face: I used to do this too, but I think it's actually better just to keep it specific and then if it needs to move into a bigger "webhooks" section or something later, it can. Thanks so much for making the changes!